src/lib/smartdc/generate-resolv

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 Joyent, Inc.
+#
+# Configure resolv.conf  if not set
+
+set -eu -o pipefail
+
+if [[ ! -e /etc/resolv.conf ]]; then
+  RESOLV=$(mktemp)
+
+  echo "# AUTOMATIC ZONE CONFIG" > "$RESOLV"
+  for i in $(mdata-get sdc:resolvers | jq -r .[]); do
```

Comment:
Is `jq` guaranteed to be installed? In lx brand zones `json` is available at `/native/usr/bin/json`.

Reply:
Oh, I see this was copied over from the script used in void. Provided that it’s guaranteed to be installed I have no objection.

Reply:
It's installed by the ebuild: https://github.com/ismell/gentoo-lx-brand-image-builder/blob/master/root/var/db/repos/local/sys-apps/sdc-vmtools/sdc-vmtools-9999.ebuild#L17